### ISSUE TITLE
One of the best plugins out there. No default open.

### ISSUE THREAD

**Issue by pchris7489** [young-developer/nppNavigateTo#59]
Hello,

This plugin is a must have, even why isn't it included by default with Notepad++.

One problem in consequence, is that there is no option to open it when opening Notepad++. I must always click on Plugins >> NavigateTo.

Or am I wrong ?

Thank you very much

**Comment by molsonkiko** [comment]
Installing the plugin automatically binds `Ctrl+,` (Control-comma) as a keyboard shortcut for opening the form, which is fast enough that IMO opening on startup is unnecessary.

That said, it wouldn't be too hard to add a configurable setting to open the form on startup.

**Comment by young-developer** [comment]
@pchris7489 I can add additional logic to existing settings option: "Keep dialog visible."

**Comment by pchris7489** [comment]
Yes thank you, as I find myself using this plugin systematically when opening Npp, me having more than 100 tabs open (working on multiple HTML documents projects, and searching active opened files is far quicker and more efficient for programming)

**Comment by pchris7489** [comment]
So any updates ?

**Comment by young-developer** [comment]
@pchris7489 I actually added changes but forgot to push. I will commit changes in a minute.

**Comment by young-developer** [comment]
@pchris7489 https://github.com/young-developer/nppNavigateTo/releases/tag/v2.7.1
